### ISSUE TITLE
Implement hand tracking using WebXR Hand Input API

### ISSUE THREAD

**Issue by FluorescentHallucinogen** [mozilla/hubs#5511]
What about implementing hand tracking using [WebXR Hand Input API](https://immersive-web.github.io/webxr-hand-input/)?

Here's how hand tracking looks like in [Hand Tracking Sample](https://immersive-web.github.io/webxr-samples/immersive-hands.html) from the Immersive Web Working Group's [WebXR Samples](https://immersive-web.github.io/webxr-samples/) on Meta Quest 2 device: https://web.dev/pwas-on-oculus-2/#hand-tracking.

**Comment by matthewbcool** [comment]
Thanks for your request!

relates to [Hand Tracking POC](https://github.com/mozilla/hubs/pull/4345),  #3689 and #2812

**Comment by elmau** [comment]
How do you imagine walking or teleporting with hand tracking?

**Comment by FluorescentHallucinogen** [comment]
@elmau Using hand gesture. See https://youtu.be/fsxH-z8qx58. :wink:

**Comment by elmau** [comment]
That looks like a really good solution. I thought of having a "virtual cellphone" to handle chat or moving around the scene.

**Comment by emclaren** [comment]
This isn't on our scheduled near-term roadmap, but we will keep our eyes on this potentially for the future. Thank you for reporting this.

**Comment by Dayk0** [comment]
Hand tracking is a very good thing. To use it daily in virtual reality applications, I must admit that it changes the use of virtual reality a lot.
It would be really nice to be able to implement it on hubs :)

**Comment by jywarren** [comment]
Agreed, for accessibility this would be really helpful, and lots of headsets are being sold without controllers now as well.

**Comment by jywarren** [comment]
There's also this "walk" hand gesture which is pretty nice, and seems intuitive esp for folks who aren't used to hand gestures:

https://youtu.be/ebM1IEn_12U

![Screenshot_20220705-110038](https://user-images.githubusercontent.com/24359/177358452-8ea2b14f-8cc5-4ea4-b4d2-89f0f0802daa.png)

**Comment by jywarren** [comment]
Would a PR be welcome for something like this, perhaps modeled on these control methods?

https://github.com/mozilla/hubs/blob/dd1443f321eecdcabc9630c273e4d0ace0de6db2/doc/index.md#wasd-to-analog2d

**Comment by HussainAther** [comment]
This is a great discussion, and I’d love to contribute to exploring hand tracking implementation for Hubs!
I looked into the Hand Tracking POC (#3689, #2812) and found that [summarize any findings here].
One approach could be to implement basic gesture recognition first (e.g., pinch for selection, swipe for navigation) before expanding into movement-based interactions.
Would it make sense to start with a prototype for gesture-based selection/manipulation first, similar to the example shown in [Meta’s WebXR hand tracking sample](https://web.dev/pwas-on-oculus-2/#hand-tracking)?
Also, are there any existing PRs in progress that attempt to integrate the WebXR Hand Input API? I’d be happy to test, provide feedback, or even contribute code.
